pkgs/development/python-modules/numba/cuda_path.patch

```diff
@@ -14,15 +14,12 @@ index b9988bc..a642680 100644
      ]
      by, libdir = _find_valid_path(options)
      return by, libdir
-@@ -35,18 +32,16 @@ def _get_libdevice_path_decision():
+@@ -35,16 +32,14 @@ def _get_libdevice_path_decision():
  
  def _nvvm_lib_dir():
      if IS_WIN32:
 -        return 'nvvm', 'bin'
 +        return 'bin',
-     elif IS_OSX:
--        return 'nvvm', 'lib'
-+        return 'lib',
      else:
 -        return 'nvvm', 'lib64'
 +        return 'lib64',
```

Comment:
Check: the paths for darwin and linux are the same now? Do you have a darwin device to verify that this works?

Reply:
Numba cut out support for CUDA on Darwin completely. I had to remove that logic to get the patch to apply again. (More accurately I rebased the patch on v0.56.4 which does not have that support). I can verify that CUDA definitely won't work, but this would not affect normal usage.

Reply:
There are macosx wheels for 0.56.4 on pypi thoug: https://pypi.org/project/numba/#files

Reply:
> platforms

`platforms = if cudaSupport then platforms.linux else platforms.unix`?

Reply:
I don't think it's necessary.

CUDA on Darwin is technically possible but rather dead in practice AFAICT. All the nixpkgs CUDA infrastructure is clearly marked for x86_64-linux so effort won't be wasted to build the CUDA version on Darwin because the dependencies won't ever try to build. I checked some other CUDA packages and they don't have this condition either.

Reply:
Riiiight, I probably meant that oh the `platforms` attribute is missing for numba, and now that this came up might be just the opportunity to amend this, if you will 🙃